scripts/generate-openapi.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Generate OpenAPI bindings for the Unleash API.
+# https://openapi-generator.tech/docs/generators/typescript-fetch
+
+set -feux
+cd "$(dirname "$0")"
+
+# URL to the generated open API spec.
+# Set the UNLEASH_OPENAPI_URL environment variable to override.
+UNLEASH_OPENAPI_URL="${UNLEASH_OPENAPI_URL:-http://localhost:4242/docs/openapi.json}"
```

Comment:
My suggestion: get source `json` into repository, run generators on pre-build, and add generated files to `.gitignore`